tezos/new_context/src/working_tree/working_tree.rs

```diff
@@ -194,7 +227,7 @@ impl Iterator for TreeWalker {
 
                     if let Some((k, node)) = iter.next() {
                         // TODO: what to do with errors here?
```

Comment:
Like you did above, please print an error here, now that we capture all output from the protocol runners it will at least show up in the logs.